### DISCUSSION TITLE
Updating Paperless to v2.13.5 doesn't work

### DISCUSSION THREAD

**Discussion by Spezialtrick** [community-scripts/ProxmoxVE#619] [Q&A]
I have just tried to update Paperless to v2.13.5 using the script. Unfortunately I get this error message:

`  cat: /opt/Paperless-ngx_version.txt: No such file or directory
 ✓ Ghostscript updated to 10.04.0
 ✓ Stopped all Paperless-ngx Services
 \ Updating to v2.13.5   
[ERROR] in line 97: exit code 0: while executing command /usr/bin/python3 manage.py migrate &> /dev/null `

How can I fix this?

**Comment by MickLesk** [comment]
Use search function and delete external Management Python packages.
